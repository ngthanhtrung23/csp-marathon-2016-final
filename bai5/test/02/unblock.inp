......
..123.
00123.
44523.
.65...
.677..
