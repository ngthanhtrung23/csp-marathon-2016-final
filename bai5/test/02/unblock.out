17
1 U 1
2 U 1
3 U 1
7 R 2
5 D 1
4 R 4
2 D 3
0 R 2
6 U 4
0 L 1
2 U 3
4 L 4
5 U 1
7 L 3
2 D 3
3 D 3
0 R 3

